extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -81,8 +89,103 @@ var WikiaSearch = {
 			searchForm.submit();
 		});
 
+	},
+	trackSearchResultClick: function(clickedElement) {
+		var queryparams = new URL(window.location).searchParams;
+		var query = queryparams.get('search') || queryparams.get('query');
+
+		if (!query) {
+			return;
+		}
+
+		var payload = {
+			searchPhrase: query,
+			clicked: {
+				type: 'article', // we don't show wikis results right now
+				id: parseInt(clickedElement.getAttribute('data-page-id')),
+				title: clickedElement.text,
+				position: parseInt(clickedElement.getAttribute('data-pos')),
+				thumbnail: !!clickedElement.getAttribute('data-thumbnail'),
```

Comment:
Is this meant to be a boolean? If so, maybe `hasThumbnail` would be more appropriate?

Reply:
names of properties are defined by data-eng team :)

Reply:
👍